cowprotocol/accounting/cost_coverage/cost_coverage_2601653.sql

```diff
@@ -0,0 +1,117 @@
+with
+purchased_eth as (
+    select
+        block_time,
+        sum(units_bought) as fees
+    from cow_protocol_ethereum.trades
+    where
+        buy_token_address in (0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2, 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee)
+        and trader in (0x84e5c8518c248de590d5302fd7c32d2ae6b0123c, 0x9008D19f58AAbD9eD0D60971565AA8510560ab41)
+        and receiver = 0xa03be496e67ec29bc62f01a428683d7f9c204930
+        and block_date between (timestamp '{{start_time}}' + interval '12' hour) and (timestamp '{{end_time}}' + interval '12' hour)
```

Comment:
Can you comment in the code why we use 12 hours here (it may be a surprising for an unsuspecting reader).